OpenRA.Mods.Common/Traits/Buildings/RepairableBuilding.cs

```diff
@@ -25,6 +25,13 @@ public class RepairableBuildingInfo : UpgradableTraitInfo, ITraitInfo, Requires<
 		public readonly int[] RepairBonuses = { 100, 150, 175, 200, 220, 240, 260, 280, 300 };
 		public readonly bool CancelWhenDisabled = false;
 
+		public readonly string IndicatorImage = "allyrepair";
+		public readonly string IndicatorSequence = "repair";
+
+		[Desc("Overrides the IndicatorPalettePrefix.")]
+		public readonly string IndicatorPalette = "";
+
+		[Desc("Suffixed by the interal repairing player name.")]
 		public readonly string IndicatorPalettePrefix = "player";
```

Comment:
I didn't think about that... lets leave it for a future pr which can include an upgrade rule.